BUILD

```diff
@@ -1 +1 @@
-python_requirements(requirements_relpath="test_docker_requirements.txt")
```

Comment:
huh kinda weird this changed?

Reply:
I found the PR and stuff but ok! I guess having consistent naming is a reason to have breaking changes!